packages/field-base/src/keyboard-mixin.d.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+/**
+ * A mixin that manages keyboard handling.
+ * It subscribes to the keyboard events while providing an actual implementation
+ * for the event handlers is left for the client (a component or another mixin).
+ */
+declare function KeyboardMixin<T extends new (...args: any[]) => {}>(base: T): T & KeyboardMixinConstructor;
+
+interface KeyboardMixinConstructor {
+  new (...args: any[]): KeyboardMixin;
+}
+
+interface KeyboardMixin {
+  /**
+   * A handler for the `keydown` event. By default, it does nothing.
+   * Override the method to implement your own behavior.
+   */
+  _onKeyDown(_event: KeyboardEvent): void;
```

Comment:
I would suggest to use `event` here:

```suggestion
  _onKeyDown(event: KeyboardEvent): void;
```

Reply:
Done.